src/main/resources/schemas/master_schema.json

```diff
@@ -1012,6 +1012,17 @@
                                                 "French"
                                               ],
                                               "pattern": "^.*$"
+                                            },
```

Comment:
Just a reminder this will need an update to the master schema pages. I'd suggest we do this at the end once through test

Reply:
Yes can do :)